### PR TITLE
typo

### PR DESCRIPTION
**Pull request by enumag**

### CODE REVIEW

File: src/Faker/Provider/Uuid.php [hunk 1]
```diff
@@ -10,7 +10,7 @@ class Uuid extends Base
      */
     public static function uuid()
     {
-        // fix for compatibility with 32bit architecture; seed range restricted to 62bit
+        // fix for compatibility with 32bit architecture; seed range restricted to 32bit
```
Comment (fzaninotto) [inline src/Faker/Provider/Uuid.php]: I don't understand this "seed range restricted to" comment. But concatenating two 32 bits random numbers does sound like 64 bits of randomness, not 32.

Comment (enumag) [reply src/Faker/Provider/Uuid.php]: As I understand it, it means that each `mt_rand` call is limited to 32 bits. Previously PHP_MAX_INT was used which produced 32 bit randomness on 32bit systems and 64 bit randomnes on 64bit systems. Two 32bit calls are used instead to produce 64bit result.

Comment (fzaninotto) [reply src/Faker/Provider/Uuid.php]: Can you please update the comment to make it more clear?

Comment (enumag) [inline src/Faker/Provider/Uuid.php]:
```suggestion
        // fix for compatibility with 32bit architecture; each mt_rand call is restricted to 32bit
        // two such calls will cause 64bits of randomness regardless of architecture
```

**Comment by fzaninotto** [comment]
Thanks!

**Pull request merged**